Feature/suscripciones panel (#2563)
mercadopago/devsite-docs: guides/subscription-plans/create-subscription-plan.en.md

# Create subscription

A subscription is an online tool that allows you to receive payments automatically and on a recurring basis. When you create a subscription plan, you define its name, price, and how often you will charge your customers. You can then share it via WhatsApp or any other social network.

Follow the steps below to create a subscription plan and start charging easily and without integration.

## Create a subscription plan

Start your setup by logging into your Mercado Pago account. Once there, go to the left side menu and click on the **Subscription Plans** option. You can also access it by [clicking here](https://www.mercadopago[FAKER][URL][DOMAIN]/subscription-plans#from-section=menu).

> If you have never created a subscription plan before, you will need to click the **Charge with a Subscription Plan** button to get started. Otherwise, if you have accessed this section previously, you will already be in the **Subscription Plans** section.

Author: ```suggestion
If you have never created a subscription plan before, you will need to click the **Charge with a Subscription Plan** button to get started. Otherwise, if you have accessed this section previously, you will already be in **Subscription Plans**.
```

![Create subscription](/images/subscription-plans/Tus-suscripciones-comenzar-EN.jpg)

Once there, click the **Create New Plan** button and fill in the required information:

> * **Name of your plan:** fill this field with the name you want to assign to the subscription plan you are creating. For example: “Dance Classes.” Keep in mind that this name is public, and the people who subscribe will be able to see it.

Reviewer: ```suggestion
* **Name of your plan:** fill this field with the name you want to assign to the subscription plan you are creating. For example: “Dance Classes”. Keep in mind that this name is public, and the people who subscribe will be able to see it.
```

> * **Price:** define the amount of money that people who subscribe will pay. Additionally, you will find the box **I prefer the customer to define it**, which allows subscribers to define the amount they want to pay for the subscription and, consequently, does not allow a fixed amount to be set for the subscription plan. If you prefer this model, click the box to select it.

Reviewer: ```suggestion
* **Price:** define the amount of money that will be paid for the subscription. Additionally, you will find the box **I prefer the customer to define it**, which allows subscribers to define the amount they want to pay for the subscription and, consequently, does not allow a fixed amount to be set for the subscription plan. If you prefer this model, click the box to select it.
```

* **Duration of the Subscription:** you can define whether your subscription plan will be active indefinitely or if it will automatically expire after a certain number of billing occurrences. Select **Unlimited** if you want it to remain active indefinitely. If you want it to end automatically, select **Limited** and define how many billing occurrences you want to be made.
* **Billing Date:** choose the day of the month when the subscription will be charged. _This option is only available for plans with monthly billing frequency_.
* **Charge Proportional Value on Enrollment Date:** If this option is activated, customers will pay a proportional amount upon enrolling for the remaining days until the first monthly charge. If deactivated, the additional amount will not be charged.
* **Reference Code:** Allows you to add a reference code to help you identify the plan in case you have several similar active plans.
* **Where do you want to redirect your customer?:** You can add a URL where the customer will be redirected once the subscription is completed, such as your website.

Once you have filled in all the fields, click **Create and Share** to finalize.

## Share a subscription plan

> Once the subscription plan is created, you can easily share its link. To do this, click **Copy Link** to paste it wherever you prefer or select one of the offered options to send it directly via WhatsApp, Facebook, X, or email.

Author: ```suggestion
Once the subscription plan is created, you can easily share its link. To do this, click **Copy Link** to paste it wherever you prefer or select one of the offered options to send it directly via WhatsApp, Facebook, X, or email.

```
